airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -40,6 +40,9 @@ class RedshiftDataOperator(BaseOperator):
     :param database: the name of the database
     :param sql: the SQL statement or list of  SQL statement to run
     :param cluster_identifier: unique identifier of a cluster
+    :param workgroup_name: name of the Redshift Serverless workgroup. Mutually exclusive with
```

Comment:
nitpick: same here

Reply:
Same here: addressed in 71bea87aff9be72086c85d99862568701e014bc4 :)